server/controllers/EventController/delete.js

```
@@ -0,0 +1,33 @@
+const { Event, EventPerson } = require('../../sequelize/models')
+
+module.exports = async (req, res, next) => {
+  try {
+    await sequelize.transaction(async transaction => {
+      const destroyCount = await Event.destroy({
+        where: {
+          id: req.params.event_id,
+        },
+        transaction,
+      });
+
+      if (destroyCount[0] === 0) {
+        if (!eventPerson) throw new ValidationError("Event was not deleted");
+      } else if (destroyCount[0] > 1) {
+        throw new ValidationError(`Only one Event should be deleted. ${updatedCount} events were deleted and rolled back.`);
+      }
```

I think I'm just bikeshedding, but I wanted to get your opinion on something if you don't mind.

Line 13:
```
if (destroyCount[0] === 0) {
```
Destroy count can be 0 for two reasons. 
1. Client gave an invalid `req.params.event_id`.
2. Server had an issue.

The first one is a 400,  but the second one is a 500. Should we be differentiating between those like the following? Or is that too much work?
```
const event = await Event.findAll(...);

if(!event.length) throw new ValidationError(...);

const destroyCount = await event[0].destroy();
if(destroyCount[0] === 0) throw new ServerError(...);
```

---

A database orm like sequelize shouldn't give 0 rows back if it encounters an issue. It should throw an exception. And I think you can rely on it to do so. I think it's fine the way it's written now.